core/src/main/resources/lib/hudson/help.jelly

```diff
@@ -22,10 +22,21 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 
-<!--
-  (?) icon linked to the specified @href
--->
-<?jelly escape-by-default='true'?>
-<a href="${href}">
-  <l:icon class="icon-help icon-sm" xmlns:l="/lib/layout"/>
-</a>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+  <st:documentation>
+    Icon linked to the specified @href
+
+    <st:attribute name="href" use="required">
+      Destination of the help icon's link
+    </st:attribute>
+    <st:attribute name="tooltip">
+      The tooltip to display
+    </st:attribute>
+    <st:attribute name="icon_size">
```

Comment:
I don't think we've ever had `_` in attribute names.

```suggestion
    <st:attribute name="iconSize">
```

Consistent with `checkUrl` and a few other camel cases.

Reply:
sure, I tried to use size first but that threw jelly stacktraces